static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -40,10 +44,16 @@ const persistResponse = (JsonResponse: () => void) => {
         JsonResponse.contentAccess.recurringContributor
     );
 
+    if ('alertAvailableFor' in JsonResponse) {
```

Comment:
I think we should change 'alertAvailableFor' to 'actionRequiredFor'.

Reply:
Sure.